packages/db/src/migrations/20240609010830_require_phrase_created_date/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - Made the column `createdAt` on table `Phrase` required. This step will fail if there are existing NULL values in that column.
+
+*/
+
+UPDATE "Phrase"
+  SET "createdAt" = COALESCE(data.timestamp, '2024-02-01')
+FROM (
+  SELECT ph.id, e.timestamp FROM "Phrase" AS ph
+  LEFT JOIN "GlossEvent" AS e ON e."phraseId" = ph.id
```

Comment:
This will set created at dates on all phrases before making it required. If there is no gloss event associated with the phrase (which will be true for all english glosses), then we fall back to february 1, which is one month before the first gloss event in the database